deployments/deploy-ssh.sh

```diff
@@ -479,7 +479,7 @@ spec:
             "--test-desc=$TEST_DESCRIPTION",
             "--test-type=$TEST_TYPE",
             "--test-tags=$TEST_TAGS",
-            "--testset-id=$TESTSET_ID",
+            "--testset-id=$DASH_UID",
```

Comment:
use one variable as `aetos-uuid` and remove `enable-dash`

Reply:
we are using this as env variable and cannot use env variable with `-`.